docs/topic_guides/evaluation/edge_overview.md

```diff
@@ -0,0 +1,54 @@
+# Edge Evaluation
+
+Once you have a trained model, you use it to generate edges (links) between entities (nodes). These edges will have a Match Weight and corresponding Probability.
+
+There are several strategies for checking whether the links in created in your pipeline perform as you want/expect.
+
+## Consider the Edge Metrics
+
+Edge Metrics measure for how links perform at an overall level.
+
+First, consider how you would like your model to perform. What is important for your use case? Do you want to ensure that you capture all possible matches? Or do you want to minimise the number of incorrectly predicted matches? Perhaps a combination of both?
```

Comment:
```suggestion
First, consider how you would like your model to perform. What is important for your use case? Do you want to ensure that you capture all possible matches (i.e. high recall)? Or do you want to minimise the number of incorrectly predicted matches (i.e. high precision)? Perhaps a combination of both?
```

I know you repeatedly direct the reader to the edge metrics but it's probably worth spelling it out here how to translate from the user need to the pertinent metric.